airflow/providers/ssh/hooks/ssh.py

```diff
@@ -257,58 +259,74 @@ def get_conn(self) -> paramiko.SSHClient:
         :rtype: paramiko.client.SSHClient
         """
         self.log.debug('Creating SSH client for conn_id: %s', self.ssh_conn_id)
-        client = paramiko.SSHClient()
 
-        if not self.allow_host_key_change:
-            self.log.warning(
-                "Remote Identification Change is not verified. "
-                "This won't protect against Man-In-The-Middle attacks"
-            )
-            client.load_system_host_keys()
+        max_time_to_wait = 10
+        for time_to_wait in self._exponential_sleep_generator(initial=1, maximum=max_time_to_wait):
```

Comment:
We should use tenacity here (we already use it elsewhere).

Reply:
okay.. let me change this to use tenacity.